code/modules/cargo/markets/market_items/weapons.dm

```diff
@@ -64,6 +64,16 @@
 	stock_max = 3
 	availability_prob = 45
 
+/datum/market_item/weapon/carpenter_hammer
+	name = "Carpenter hammer"
+	desc = "When you really want to look like a psycho..."
+	item = /obj/item/carpenter_hammer
+
+	price_min = CARGO_CRATE_VALUE * 1
+	price_max = CARGO_CRATE_VALUE * 1.25
+	stock_max = 2
+	availability_prob = 65
```

Comment:
I have concerns about the availability of this item, as you're offering it at less than the chainsaw's cost, at higher availability, and higher stock, for an arguably higher offensive weapon. For reference, here's the chainsaw's market_item qualities.
```
	price_min = CARGO_CRATE_VALUE * 1.75 //350 credits
	price_max = CARGO_CRATE_VALUE * 3 //600 credits
	stock_max = 1
	availability_prob = 35```
	
I would want it to be either a higher price (Maybe 400-800 credits), or a lower availability/stock before I ran with this.

Reply:
I think you're overshooting it. It's better to have something that's not underpowered out of mere prudence on release. That way we get people to try it out and it gives us actual feedback.

Reply:
The way it's sitting right now, it's also less expensive than every other weapon on the black market, as a 20 force, normal sized melee weapon, WITH a demolition_mod of 25%. I'm not suggesting nerfing it into the ground, just that it's starting out very high on the power scale.

Reply:
All good, I simply meant to say that the values you were suggesting seemed a smidge too tall, especially the availability.

Reply:
the stats are undertuned on every other black market item which comparatively makes this look overtuned